src/area.c

```diff
@@ -102,8 +115,9 @@ LIBUI_FUNCTION(create) {
 
 	status = napi_create_reference(env, handle_external, 1, &(area->area_ref));
 	CHECK_STATUS_THROW(status, napi_create_reference);
+	*/
 
-	return handle_external;
+	return control_handle_new(env, uiControl(ui_area), "area");
```

Comment:
Current implementation of all containers expects to receive an `[External]` that point to a [struct control_handle](https://github.com/parro-it/libui-napi/blob/f49bae7cbdc317226e56298d648bdc4434c9b1de/src/includes/control.h#L31).
`control_handle_new` create such data structure given a pointer to `struct uiControl`.

Reply:
I did it this way to because the handler callbacks need to be stored somewhere.

Reply:
Oh yes, see my other comments below...